modules/dgi_actions_ark_identifier/src/Plugin/Action/MintArkIdentifier.php

```diff
@@ -103,7 +104,10 @@ protected function getIdentifierFromResponse(Response $response): string {
         '@id' => $this->getEntity()->id(),
         '@contents' => $contents,
       ]);
-      return "{$this->getIdentifier()->getServiceData()->getData()['host']}/id/{$response['success']}";
+      $ark = $response['success'];
+      $service_data = $this->getIdentifier()->getServiceData()->getData();
+      $resolver = (array_key_exists('resolver', $service_data)) ? $service_data['resolver'] : "{$service_data['host']}/id";
```

Comment:
Think this might need to be a little more robust between here and the admin form for the `service_data` entity. That is the inclusion of the `array_key_exists` handles previously created config entities which is 👍, however if one was to go back and save the admin form as it were it would then store an empty `resolver` value in the configuration.

Example:
```
label: 'CDL EZID'
id: cdl_ezid
service_data_type: ezid
data:
  host: 'https://ezid.cdlib.org'
  namespace: 'ark:/99999/fk4'
  resolver: ''
  ```
  
  When it comes time to process this as the key exists it'll use the empty string and eventually die out:
  ```
 InvalidArgumentException: The URI '/ark:/99999/fk4ht44x2h' is invalid. You must use a valid URI scheme. in Drupal\Core\Url::fromUri() (line 291 of core/lib/Drupal/Core/Url.php). 
 ```